client/me/purchases/remove-plan-dialog/index.jsx

```diff
@@ -0,0 +1,61 @@
+import { Dialog } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component, Fragment } from 'react';
+import FormSectionHeading from 'calypso/components/forms/form-section-heading';
+
+import './style.scss';
+
+class RemovePlanDialog extends Component {
```

Comment:
IMO it's better to use function components instead of class components. For one it's less code/boilerplate all round and its much simpler to read.

Reply:
component refactored in the file https://github.com/Automattic/wp-calypso/pull/67090/files#diff-22f6c96a70e669c1ee86915b052fe607759a7d300568afe8a92d9646c37860d0